subsys/bluetooth/mesh/sar_cfg_srv.c

```diff
@@ -135,6 +159,41 @@ static int sar_cfg_srv_init(struct bt_mesh_model *model)
 	return 0;
 }
 
+static void sar_cfg_srv_reset(struct bt_mesh_model *model)
+{
+	struct bt_mesh_sar_tx sar_tx = BT_MESH_SAR_TX_INIT;
+	struct bt_mesh_sar_rx sar_rx = BT_MESH_SAR_RX_INIT;
+
+	memcpy(&bt_mesh.sar_tx, &sar_tx, sizeof(struct bt_mesh_sar_tx));
+	memcpy(&bt_mesh.sar_rx, &sar_rx, sizeof(struct bt_mesh_sar_rx));
```

Comment:
Assignment operator can be used for structs.

Reply:
You are of course right, my bad